common/locales/en/defaultTasks.json

```diff
@@ -1,21 +1,19 @@
 {
-  "defaultHabit1Text": "Productive Work (Click the pencil to edit)",
```

Comment:
Looks like your editor may have auto-formatted this file. Please revert that change.